### PR TITLE
add unit test for Medkumo sdk

### PR DESCRIPTION
**Pull request by taibuitan**

### CODE REVIEW

**Comment by taibuitan** [comment]
Hi @nguyentannghia 
Please review my code.
Thanks

File: test/index.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<html>
+
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width">
+    <title>QUnit Example</title>
```
Comment (tailehuu) [inline test/index.html:6]: Medkumo Testing

File: test/index.html [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    <script src="http://sdk.medkumo.loc/test/lib/qunit-2.0.1.js"></script>
+    <script>
+        (function() {
+            var script = document.createElement('script');
+            script.async = true;
+            script.src = 'http://sdk.medkumo.loc/medkumo.js';
+            var entry = document.getElementsByTagName('script')[0];
+            entry.parentNode.insertBefore(script, entry);
+        })();
+    </script>
```
Comment (tailehuu) [inline test/index.html:22]: add this code before </script>


window.Medkumo_ready = function() {
    Medkumo.init('<your hospital key>', 'test');
  };

File: test/js/appointment.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
+function loadMedkumoSDK() {
```
Comment (tailehuu) [inline test/js/appointment.js:1]: remove  this function

File: test/js/appointment.js [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    Medkumo.init('<your hospital key>', 'test');
+}
+
+QUnit.test("Should display the book appointment form when clicking book an appointment button", function(assert) {
+    loadMedkumoSDK();
```
Comment (tailehuu) [inline test/js/appointment.js:6]: remove this line

File: test/js/appointment.js [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+    var done = assert.async();
+    setTimeout(function() {
+        assert.equal(1, 1);
+        done();
+    }, 200);
+});
+
+QUnit.test("Should display successful message when clicking book button ", function(assert) {
+    loadMedkumoSDK();
```
Comment (tailehuu) [inline test/js/appointment.js:15]: same here

File: test/js/appointment.js [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+    var done = assert.async();
+    setTimeout(function() {
+        assert.equal(1, 1);
+        done();
+    }, 200);
+});
+
+QUnit.test("Should display error message when clicking book button ", function(assert) {
+    loadMedkumoSDK();
```
Comment (tailehuu) [inline test/js/appointment.js:24]: same here.

File: test/js/doctor.js [hunk 7]
```diff
@@ -0,0 +1,21 @@
+function loadMedkumoSDK() {
```
Comment (tailehuu) [inline test/js/doctor.js:1]: remove this

**Pull request closed without merging**